lib/tbd/psi.rb

```diff
@@ -1717,49 +1717,25 @@ def process(model = nil, argh = {})
         t_model.wires.each do |wire|
           next unless surface[:wire] == wire.id # should be a unique match
 
-          normal     = tbd[:surfaces][id][:n]   if tbd[:surfaces].key?(id)
-          normal     = holes[id].attributes[:n] if holes.key?(id)
-          normal     = shades[id][:n]           if shades.key?(id)
-          farthest   = Topolys::Point3D.new(origin.x, origin.y, origin.z)
-          farthest_V = farthest - origin # zero magnitude, initially
-          inverted   = false
-          i_origin   = wire.points.index(origin)
-          i_terminal = wire.points.index(terminal)
-          i_last     = wire.points.size - 1
-
-          if i_terminal == 0
-            inverted = true unless i_origin == i_last
-          elsif i_origin == i_last
-            inverted = true unless i_terminal == 0
-          else
-            inverted = true unless i_terminal - i_origin == 1
-          end
```

Comment:
A skylight well cutout (or any other cutout) + leader lines combo breaks any expectation that Topolys wire vertex indices follow sequentially, e.g. [0, 1, 2, 3, 4]. With cutouts/leader lines (which need to backtrack), one could encounter e.g. [0, 1, 2, 3, 4, 2, 3, etc.]. For some edges, this would unfortunately break the _inverted_ test, which would subsequently screw up polar vectors and subsequent angle calculations.

All things considered, easier to scrap the test altogether than fix it. In fact the test is no longer warranted given the polygon checks upstream.